Extensions/EdgeScrollCamera.json

```diff
@@ -0,0 +1,1473 @@
+{
+  "author": "",
+  "category": "General",
+  "description": "How to use:\n- Run extension every frame you want the camera to move\n\nTips:\n- Camera movement speed is constant, even when traveling in diagonal directions",
+  "extensionNamespace": "",
+  "fullName": "Edge scroll camera",
+  "helpPath": "",
+  "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLWNhbWVyYS1tZXRlcmluZy1tYXRyaXgiIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNNCw0SDIwQTIsMiAwIDAsMSAyMiw2VjE4QTIsMiAwIDAsMSAyMCwyMEg0QTIsMiAwIDAsMSAyLDE4VjZBMiwyIDAgMCwxIDQsNE00LDZWMThIMjBWNkg0TTUuNSw3LjVIMTFWOS4xN0MxMC4xNSw5LjQ3IDkuNDcsMTAuMTUgOS4xNywxMUg1LjVWNy41TTE4LjUsNy41VjExSDE0LjgzQzE0LjUzLDEwLjE1IDEzLjg1LDkuNDcgMTMsOS4xN1Y3LjVIMTguNU0xOC41LDE2LjVIMTNWMTQuODNDMTMuODUsMTQuNTMgMTQuNTMsMTMuODUgMTQuODMsMTNIMTguNVYxNi41TTUuNSwxNi41VjEzSDkuMTdDOS40NywxMy44NSAxMC4xNSwxNC41MyAxMSwxNC44M1YxNi41SDUuNU0xMiwxMC41QTEuNSwxLjUgMCAwLDEgMTMuNSwxMkExLjUsMS41IDAgMCwxIDEyLDEzLjVBMS41LDEuNSAwIDAsMSAxMC41LDEyQTEuNSwxLjUgMCAwLDEgMTIsMTAuNVoiIC8+PC9zdmc+",
+  "name": "EdgeScrollCamera",
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/camera-metering-matrix.svg",
+  "shortDescription": "Scroll camera when mouse is near edge of screen.",
+  "version": "0.1.0",
+  "tags": [
+    "camera",
+    "scroll",
+    "edge",
+    "screen"
+  ],
+  "authorIds": [
+    "gqDaZjCfevOOxBYkK6zlhtZnXCg1"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [
+    {
+      "description": "Scroll camera when mouse is near edge of screen.",
+      "fullName": "Scroll camera when mouse is near edge of screen",
+      "functionType": "Action",
+      "group": "",
+      "name": "EdgeScrollCamera",
+      "private": false,
+      "sentence": "Scroll camera when mouse is near edge of screen (Layer: _PARAM9_, Trigger width: Top _PARAM1_ px, Bottom _PARAM2_ px, Left _PARAM3_ px, Right _PARAM4_ px, Move speed: Up _PARAM5_ px/second, Down _PARAM6_ px/second, Left _PARAM7_ px/second, Right _PARAM8_ px/second)",
```

Comment:
Lot of option, the sentence is so long, might be possible to hide some parameters?

Reply:
Do you think people will want different borders and speeds per side?  In most games I have played, those values are the same on all 4 sides.

Reply:
One value for speed is fine yeah.